workflow/views/time_overview_view.py

```diff
@@ -32,6 +32,8 @@ def get(self, request, start_date=None, *args, **kwargs):
         staff_data = []
         all_staff = Staff.objects.all()
         for staff_member in all_staff:
+            if staff_member.is_staff is True:
```

Comment:
This is a perfect example of where comments are important.

Something like "App users are not workers in the factory and so should not be having timesheet entries"